c2corg_api/models/report.py

```diff
@@ -0,0 +1,224 @@
+    #
+
+    # Etude de l'itinéraire
+    # - Study route
+    route_study = Column(String)
+
+    # Etude des conditions
+    # - Study conditions
+    conditions = Column(String)
+
+    # Préparation physique et entraînement par rapport à l'objectif
+    # - physical preparation and training in relation to the objective
+    training = Column(String)
+
+    # Motivations
+    # - Motivations
+    motivations = Column(String)
+
+    # Dynamique de groupe
+    # - Group dynamic
+    xdyn = Column(String)
+
+    # IS IT? group_management
+
+    # Niveau d'attention et (ré)évaluation des risques
+    # - Attentiveness and (revaluation risk)
+    risk = Column(String)
+
+    # Gestion du temps
+    # - Management of time
+    time_management = Column(String)
+
+    # Mesures et techniques de sécurité mises en oeuvres
+    # - Safety measures and technical implementations
+    safety = Column(String)
+
+    # Eléments qui ont atténué les conséquences
+    # - Elements that have mitigated the impact
+    xelem = Column(String)
+
+    # Conséquences sur ses pratiques
+    # - Its practical consequences
+    xcons_prac = Column(String)
+
+    # Conséquences physiques et autres commentaires
+    # - physical consequences and other comments
+    xcons_phys = Column(String)
+
+    # reduce_impact
+    # increase_impact
+    # modifications
+    # other_comments
+
+
+attributes_locales = [
+    'route_study', 'conditions', 'training', 'motivations', 'xdyn', 'risk',
+    'time_management', 'safety', 'xelem', 'xcons_prac', 'xcons_phys',
+]
+
+
+class Report(_ReportMixin, Document):
+    """
+    """
+    __tablename__ = 'reports'
+
+    document_id = Column(
+        Integer,
+        ForeignKey(schema + '.documents.document_id'), primary_key=True)
+
+    __mapper_args__ = {
+        'polymorphic_identity': REPORT_TYPE,
+        'inherit_condition': Document.document_id == document_id
+    }
+
+    def to_archive(self):
+        x = ArchiveReport()
+        super(Report, self)._to_archive(x)
+        copy_attributes(self, x, attributes)
+
+        return x
+
+    def update(self, other):
+        super(Report, self).update(other)
+        copy_attributes(other, self, attributes)
+
+
+class ArchiveReport(_ReportMixin, ArchiveDocument):
+    """
+    """
+    __tablename__ = 'reports_archives'
+
+    id = Column(
+        Integer,
+        ForeignKey(schema + '.documents_archives.id'), primary_key=True)
+
+    __mapper_args__ = {
+        'polymorphic_identity': REPORT_TYPE,
+        'inherit_condition': ArchiveDocument.id == id
+    }
+
+    __table_args__ = Base.__table_args__
+
+
+schema_report = SQLAlchemySchemaNode(
+    Report,
+    # whitelisted attributes
+    includes=schema_attributes + attributes,
+    overrides={
+        'document_id': {
+            'missing': None
+        },
+        'version': {
+            'missing': None
+        },
+        'locales': {
+            'children': [schema_document_locale]
```

Comment:
You will need a custom schema for the locales (`schema_report_locale`). Like here: https://github.com/c2corg/v6_api/blob/master/c2corg_api/models/route.py#L290